src/UIComponents/DataDisplay.jsx

```diff
@@ -25,7 +25,7 @@ class DataDisplay extends Component {
           <DataTable />
         </div>
         <div style={styles.footerText}>
-          There are {this.props.data.length} rows of data.
+          There are {this.props.data.length} rows of data. If rows contained empty cells, they were removed.
```

Comment:
I think we should say something like "Another X rows contained empty cells and were removed." and only show it if we actually removed rows.

Reply:
(The guiding principle here is that if the software knows the answer, in this case whether there were empty cells, then it can be specific about what it tells the user.)

Reply:
Updated code @ screenshots! Thanks for suggesting.